syntax = "proto3";

message BaseMessage {
    string type = 1;
}

message SysMessage {
    string type = 1;
    int64 used_memory = 2;
    int64 total_memory = 3;
    float cpu_usage = 4;
    float ssd_usage = 5;
}